src/test/java/tests/RegistrationWithErrorsTest.java

```diff
@@ -13,15 +14,23 @@ public class RegistrationWithErrorsTest extends TestBase {
     RegistrationPage registerObj;
     String fn = loadPropertiesFiles.userData.getProperty("firstName");
     String ln = loadPropertiesFiles.userData.getProperty("lastName");
+    Faker fakeData = new Faker();
+    String email = fakeData.internet().emailAddress();
     String tel = loadPropertiesFiles.userData.getProperty("telephone");
-    String password = loadPropertiesFiles.userData.getProperty("password");
+    String invalidPassword = loadPropertiesFiles.userData.getProperty("inValidPassword");
 
     @Test
     public void invalidRegistration () throws InterruptedException {
         homeObject = new HomePage(driver);
         homeObject.clickOnRegisterBtn();
         registerObj = new RegistrationPage(driver);
-        registerObj.InvalidRegistration(fn, ln, tel, password);
+        registerObj.InvalidRegistrationWithFN_LN(fn, ln);
         Assert.assertTrue(driver.findElement(registerObj.emailValidation_error).isDisplayed());
```

Comment:
don't use driver.findelement in the test use it in the pageobject
create a boolean method that returns try or false in the page object with parameter, then call the method in the test and use the parameter